### PR TITLE
added search option by dappID

### PR DESCRIPTION
**Pull request by rishikesh-y**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# package.json is formatted by package managers, so we ignore it here
+# package.on is formatted by package managers, so we ignore it here
```
Comment (r4881t) [inline .prettierignore:1]: Why?

File: src/lib/registry.ts [hunk 2]
```diff
@@ -203,7 +217,7 @@ export class DappStoreRegistry {
         debug("registry changed. updating...");
         this.cachedRegistry = remoteRegistry;
         this.lastRegistryCheckedAt = new Date();
-        this.searchEngine.addDocuments(this.cachedRegistry.dapps);
+        // this.searchEngine?.addAll(this.cachedRegistry.dapps);
```
Comment (r4881t) [inline src/lib/registry.ts:220]: Index should also happen here

File: src/lib/registry.ts [hunk 3]
```diff
@@ -177,7 +191,7 @@ export class DappStoreRegistry {
           );
           break;
       }
-      this.searchEngine.addDocuments(this.cachedRegistry.dapps);
+      // this.searchEngine?.addAll(this.cachedRegistry.dapps);
```
Comment (r4881t) [inline src/lib/registry.ts:194]: Index should also happen here

**Review by r4881t — COMMENTED** [review]
- [ ] Increment the package version in package.json
- [ ] Update the `storekit` repo to add a controller corresponding to search by dappId

File: src/lib/registry.ts (hunk 2 above)
Comment (rishikesh-y) [reply src/lib/registry.ts:220]: if we add same document more than one time, than it is throwing an error. indexing already happening in buildSearchIndex function after calling registry.

File: src/lib/registry.ts (hunk 3 above)
Comment (rishikesh-y) [reply src/lib/registry.ts:194]: same as above

File: src/lib/registry.ts (hunk 2 above)
Comment (r4881t) [reply src/lib/registry.ts:220]: ok

**Comment by rishikesh-y** [comment]
i will update the storeKit repo to add a controller corresponding to search by dappId. Once this will get merge

File: .prettierignore (hunk 1 above)
Comment (r4881t) [reply .prettierignore]: Undo the spelling issue

Comment (rishikesh-y) [reply .prettierignore]: done that already

**Review by r4881t — APPROVED** [review]

**Pull request merged**